L1Trigger/Phase2L1ParticleFlow/python/l1ctLayer1_cff.py

```diff
@@ -456,72 +456,72 @@
 )
 
 
-l1ctLayer1 = cms.EDProducer("L1TPFCandMultiMerger",
+l1tLayer1 = cms.EDProducer("L1TPFCandMultiMerger",
     pfProducers = cms.VInputTag(
-        cms.InputTag("l1ctLayer1Barrel"),
-        cms.InputTag("l1ctLayer1HGCal"),
-        cms.InputTag("l1ctLayer1HGCalNoTK"),
-        cms.InputTag("l1ctLayer1HF")
+        cms.InputTag("l1tLayer1Barrel"),
+        cms.InputTag("l1tLayer1HGCal"),
+        cms.InputTag("l1tLayer1HGCalNoTK"),
+        cms.InputTag("l1tLayer1HF")
     ),
     labelsToMerge = cms.vstring("PF", "Puppi", "Calo", "TK"),
     regionalLabelsToMerge = cms.vstring("Puppi"),
 )
 
-l1ctLayer1EG = cms.EDProducer(
+l1tLayer1EG = cms.EDProducer(
     "L1TEGMultiMerger",
     tkElectrons=cms.VPSet(
         cms.PSet(
             instance=cms.string("L1TkEleEE"),
             pfProducers=cms.VInputTag(
-                cms.InputTag("l1ctLayer1HGCal", 'L1TkEle')
+                cms.InputTag("l1tLayer1HGCal", 'L1TkEle')
             )
         ),
         cms.PSet(
             instance=cms.string("L1TkEleEB"),
             pfProducers=cms.VInputTag(
-                cms.InputTag("l1ctLayer1Barrel", 'L1TkEle')
+                cms.InputTag("l1tLayer1Barrel", 'L1TkEle')
             )
         )
     ),
     tkEms=cms.VPSet(
         cms.PSet(
             instance=cms.string("L1TkEmEE"),
             pfProducers=cms.VInputTag(
-                cms.InputTag("l1ctLayer1HGCal", 'L1TkEm'),
-                cms.InputTag("l1ctLayer1HGCalNoTK", 'L1TkEm')
+                cms.InputTag("l1tLayer1HGCal", 'L1TkEm'),
+                cms.InputTag("l1tLayer1HGCalNoTK", 'L1TkEm')
             )
         ),
         cms.PSet(
             instance=cms.string("L1TkEmEB"),
             pfProducers=cms.VInputTag(
-                cms.InputTag("l1ctLayer1Barrel", 'L1TkEm')
+                cms.InputTag("l1tLayer1Barrel", 'L1TkEm')
             )
         )
     ),
     tkEgs=cms.VPSet(
         cms.PSet(
             instance=cms.string("L1EgEE"),
             pfProducers=cms.VInputTag(
-                cms.InputTag("l1ctLayer1HGCal", 'L1Eg'),
-                cms.InputTag("l1ctLayer1HGCalNoTK", 'L1Eg')
+                cms.InputTag("l1tLayer1HGCal", 'L1Eg'),
+                cms.InputTag("l1tLayer1HGCalNoTK", 'L1Eg')
             )
         )    
     )
 )
 
-l1ctLayer1TaskInputsTask = cms.Task(
+L1TLayer1TaskInputsTask = cms.Task(
     pfClustersFromL1EGClusters,
     pfClustersFromCombinedCaloHCal,
     pfClustersFromCombinedCaloHF,
     pfClustersFromHGC3DClusters,
     pfTracksFromL1Tracks
```

Comment:
Shouldn't these follow the naming convention? Judging from the name of the Task, they are L1T modules.

Reply:
Done